libbeat/idxmgmt/ilm/client_handler.go

```diff
@@ -21,15 +21,18 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/url"
+	"os"
 	"path"
 
 	"github.com/elastic/beats/libbeat/common"
 )
 
 type esClientHandler struct {
-	client ESClient
+	client Client
 }
 
+type stdoutClientHandler struct{}
```

Comment:
I see. Maybe we want it not be stdout only, but have a constructor users can pass a writer too.

Is there a chance we need to configure elements of the handler? e.g. the export commands allow us to set ES/Kibana version we want to target via CLI flags.

Reply:
I changed the handling in how the handler are passed in. The change now simply works with `FileClientHandler` which allows to `export` tempates and policies also to a file and not only to stdout.